src/Raven.Server/Documents/Indexes/Static/AcornimaVisitor.cs

```diff
@@ -85,6 +85,30 @@ public virtual void VisitStatement(Statement statement)
                 case NodeType.CatchClause:
                     VisitCatchClause(statement.As<CatchClause>());
                     break;
+                case NodeType.ChainExpression:
+                    VisitChainExpression(statement.As<ChainExpression>());
+                    break;
+                case NodeType.TemplateLiteral:
+                    VisitTemplateLiteral(statement.As<TemplateLiteral>());
+                    break;
+                case NodeType.ForOfStatement:
+                    VisitForOfStatement(statement.As<ForOfStatement>());
+                    break;
+                case NodeType.ClassDeclaration:
+                    VisitClassDeclaration(statement.As<ClassDeclaration>());
+                    break;
+                case NodeType.ImportDeclaration:
+                    VisitImportDeclaration(statement.As<ImportDeclaration>());
+                    break;
+                case NodeType.ExportNamedDeclaration:
+                    VisitExportNamedDeclaration(statement.As<ExportNamedDeclaration>());
+                    break;
+                case NodeType.ExportAllDeclaration:
+                    VisitExportAllDeclaration(statement.As<ExportAllDeclaration>());
+                    break;
+                case NodeType.ExportDefaultDeclaration:
+                    VisitExportDefaultDeclaration(statement.As<ExportDefaultDeclaration>());
+                    break;
```

Comment:
What happens if `import` and `export` are used in indexes? Is there any import resolution behind the scenes? This could be extremely useful if it's possible to install helper packages, e.g. `date-fns` for handling dates in the index.